pom.xml

```diff
@@ -71,6 +71,11 @@
           <artifactId>maven-release-plugin</artifactId>
           <version>2.5</version>
         </plugin>
+        <plugin>
+          <groupId>org.jenkins-ci.tools</groupId>
+          <artifactId>maven-hpi-plugin</artifactId>
+          <version>1.95</version>
```

Comment:
Probably because this is the current one :)
https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin